RELEASE.md

```diff
@@ -5,20 +5,33 @@ Please follow the established format:
 - Use present tense (e.g. 'Add new feature')
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
-# Upcoming Release
+# Release 10.0.0
 
 ## Major features and improvements
 
-- Introduce `onActionCallback` prop in Kedro-Viz react component. (#2022)
-- Introduce `kedro viz --lite`, which allows users to run Kedro-Viz without installing Kedro project dependencies. (#1966, #2077)
-- Slice a pipeline functionality. (#2036)
+- Add `kedro viz --lite`, allowing users to run Kedro-Viz without installing Kedro project dependencies. (#1966, #2077, #2093)
+- Enable visual slicing of a Kedro pipeline in Kedro-viz. (#2036)
+- Improve Kedro-Viz CLI startup time with lazy subcommands and deferring imports (#1920)
+- Add documentation for Kedro-viz in VSCode Extension. (#2078)
```

Comment:
Im wondering if this one should be in the "other changes" as i see other documentation work is mentioned there too?

Reply:
I added this here, because Kedroviz on VsCode is a big feature. So highlighting it here as a main feature although most of the code resides in vscode-kedro repo